README.md

```diff
@@ -145,25 +146,25 @@ Please examine the XML docs for full details. However, in simplified form, we ha
   * Provide a factory method, to set a property to a newly created value each time.
 * ##### `WithBuilder()`
   * Like `WithFactory`, but the factory is specifically the `.Build()` method of the provided `Builder`.
-* ##### `WithBuilt()`
-  * Like `WithBuilder`, but the appropriate `Builder<T>` is found automatically from any classes tagged with `BuilderFactory`, so you don't need to provide any second argument at all ... it just works it all out.
 * ##### _`IEnumerable` variations of `WithFactory()`_
```

Comment:
Do these still exist? Or are they subsumed by `WithCreateEnumerableFrom`?

Reply:
Yes, this exists and is distinct from `WithCreateEnumerableFrom`.